mechanical/assembly_renderer.py

```diff
@@ -16,17 +16,19 @@
 """
 
 import os
+import logging
 from pathlib import Path
 import cadquery as cq
 import yaml
-
 from nimble_build_system.cad.shelf import create_shelf_for
-
 from nimble_build_system.cad.rack_assembly import RackAssembly
+# from nimble_build_system.orchestration.paths import ABS_PATH
 
 assembly_definition_file = "../build/assembly-def.yaml"
 render_destination = os.path.join(os.getcwd(), "renders")
 
+logging.info("RENDER: %s", render_destination)
```

Comment:
It would be best to make a logger with `getLogger(__name__)` rather than use the root logger